content/docs/user-guide/remote-content/use-a-remote-repository.md

```diff
@@ -0,0 +1,163 @@
+
+1. If you have a URL for an online Gem repository, copy and paste it into the `Repository Path` field and press the **Add** button.
+1. Alternatively, if you want to add an O3DE remote repository from your local hard drive, enter a local path on your computer or press the folder button on the right side of the Repository Path field to open the `Browse` dialog box, select the folder with the O3DE remote repository, press the **Select Folder** button, and finally press the **Add** button.
+
+
+A new entry will be added to the list of remote sources, and the Gems, Projects and Templates provided by the O3DE remote repository will be available to download and use.
+
+### Activate or deactivate a remote repository
+
+You can toggle the visibility of remote content from a O3DE remote repository by pressing the visibility button on the same row as the O3DE remote repository on the **Remote Sources** page.  This is a useful way to hide content from that repository, without removing the remote repository.
+
+![Visibility buttons on the Remote Sources page](/images/user-guide/remote-content/ProjectManagerRemoteSources.JPG)
+
+### Remove a remote repository
+
+You can remove an O3DE remote repository URI from the **Remote Sources** page in the Project Manager so the remote content in it is no longer listed and the meta data is no longer updated.
+1. Select the O3DE remote repository you want to remove by clicking on the entry in the list of remote sources.
+2. Press the **Remove** button in the right panel.  You will be asked to confirm if you want to remove the remote repository.
+
+### Download Gems from a remote repository
+
+You can download Gems from the **Gems** tab in the Project Manager by clicking on the row of the Gem you want to download and then pressing the **Download Gem** button in the right pane.  
+
+![Visibility buttons on the Remote Sources page](/images/user-guide/remote-content/ProjectManagerDownloadGem.JPG)
+
+Gems available in remote repositories will also be downloaded automatically when activated in a project from the **Gem Catalog** page which can be accessed by pressing the **Configure Gems** button on the Edit Project Settings page for a project, or from the **Configure Gems...** project menu option on the **Projects** tab.
+
+### Download projects from a remote repository
+
+You can download projects from the **Projects** tab in the Project Manager by pressing the **Download** button on the project you want to download.  
+
+### Download templates from a remote repository
+
+You can download templates from the **Create a New Project** page in the Project Manager by selecting the project template in the list of project templates and pressing the  **Download Template** button in the right pane.  
+
+
+## Using the `o3de` command line tool to access O3DE remote repositories 
+
+You can use the `o3de` command line tool to register, refresh, activate and deactivate remote repositories and download remote content.
+
```

Comment:
Should we include an example for this like the other commands?

Reply:
The examples are in the sub sections, but I'll rephrase this and add a link to the official CLI docs